compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1501,6 +1503,11 @@ class Allocator {
                 getPageSizeCached()),
         "scudo:ring_buffer");
```

Comment:
Shouldn't this call include the `MAP_ALLOWNOMEM` flag? (it should be possible to get a `nullptr` if you don't pass this flag - if you've seen one without it, there might some hidden bug elsewhere)

Reply:
Oh, and you may want to just check the return value of `map` (it returns a bool) instead of testing if `getBase()` is a nullptr later

Reply:
Ah! I missed that. I didn't actually see that, just wanted to add this as a precaution. So I think this CL is not needed.